tests/integration/test_interaction.py

```diff
@@ -144,21 +130,60 @@ def get_locations():
                 input_div = self._get_input_div_by_value(item_key)
                 self.wait_until_has_class('correct', input_div)
 
-        self.wait_until_html_in(self.feedback['final'], self._get_feedback_message())
+        self.wait_until_html_in(feedback['final'], self._get_feedback_message())
 
         # Scroll "Reset exercise" button into view to make sure Selenium can successfully click it
-        self.scroll_down(pixels=250)
+        self.scroll_down(pixels=scroll_down+150)
 
-        reset = self._page.find_element_by_css_selector('.reset-button')
+        reset = self.get_reset_button()
         reset.click()
 
-        self.wait_until_html_in(self.feedback['intro'], self._get_feedback_message())
+        self.wait_until_html_in(feedback['intro'], self._get_feedback_message())
 
         locations_after_reset = get_locations()
         for item_key in items.keys():
             self.assertDictEqual(locations_after_reset[item_key], initial_locations[item_key])
 
 
+class InteractionTestFixture(InteractionTestBase):
+    """
+    Verifying Drag and Drop XBlock rendering against default data - if default data changes this would probably broke
```

Comment:
@e-kolpakov Can you change this to:

```
Verifying Drag and Drop XBlock rendering against default data - if default data changes this will probably break.
```

That would make the docstring consistent with the one in [test_render.py](https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/master/tests/integration/test_render.py#L26-L27).